assets/PearClubs.txt

```diff
@@ -0,0 +1,537 @@
+Cornellians for Israel
+Cornell University Chess Club
+Climate Justice Cornell
+Coalition of Pan-African Scholars
+Cornell College Republicans
+Computer Animation Club
+Consent Ed
+Cornell Dance Dance Revolution Club
+The Cornell University Democrats
+Cornell University Dietetics Association
+Duel Links at Cornell
+Cornell University Emergency Medical Services
+Cornell Environmental Collaborative
+Esports at Cornell
+Cornell European Business Society
+Flow Club
+Friends of Gorge
+Games Club at Cornell
+Guild of Visual Arts
+Cornell Hammock Enthusiasts’ Club
+Cornell Health International
+Her Campus at Cornell
+Hip-Hop Heads
+Hortus Forum
+ILR Women’s Caucus
+International Students Union
+Cornell Japanese Animation Society
+The Johnson Museum Club
+Language Expansion Program
+Les Femmes de Substance
+Libertarians at Cornell
+Mapping Society at Cornell
+Media Guild, Inc
+The Millennial Voices Project
+Cornell Minds Matter
+Mock Trial Association
+Movement Exchange at Cornell
+Cornell Nature Society
+NatureRx
+Naval ROTC
+NCPA Paintball Club
+NEO@CU - (Northeast Ohio at Cornell University)
+Peace Action Cornell
+Period @ Cornell
+Phoenix Society
+Cornell Photo Society
+Planned Parenthood Generation Action
+The Pokemon League
+Cornell Political Union
+Prison Reform and Education Project (PREP)
+Project Greenhouse
+Cornell Quiz Bowl Club
+Quodlibet
+Cornell Raptor Program
+Rewriting the Code at Cornell
+Cornell Roosevelt Institute
+Scabbard & Blade Society
+Scheinman Conflict Resolution Club at Cornell
+Scholars Working Ambitiously to Graduate
+Science Olympiad at Cornell
+Snodgrass and Wigglesworth, Cornell Undergraduate Entomology Club
+Society for Natural Resources Conservation
+Society for the Promotion of Indian Classical Music And Culture Amongst Youth
+Student Educators: Art & the Museum
+Cornell Students for Animal Rights
+Students Off Booze Enjoying Recovery
+Von Cramm Cooperative Hall
+Wait Avenue Cooperative
+Wait Terrace Cooperative
+Watermargin Cooperative
+Wildlife Society, Cornell University Student Chapter
+Women of Color Coalition
+Women’s Planning Forum
+Women in Public Policy
+Women's Resource Center, Cornell
+Cornell Young Democratic Socialists
+Cornell University Class Council
+A Capella Advisory Council (ACAC)
+Club Sports Council
+Community Partnership Funding Board
+Cornell Concert Commission (CCC)
+Cornell University Program Board (CPUB)
+CUTonight Commission
+Dyson Undergraduate Council
+Fanclub Collective
+Fantastic Fungi Fanatics
+Farmers’ Market at Cornell
+Interfraternity Council
+Ivy Council
+Jewish Greek Council (JGC)
+Men of Color Council
+Multicultural Concert Funding and Advisory Board
+Multicultural Greek Letter Cornell
+National Residence Hall Honorary
+Orientation Steering Committee
+Peer Review Board
+Residential Student Congress
+Risley Arts Subcommittee
+Slope Day Programming Board
+South Asian Council
+Student Athlete Advisory Committee
+Willard Straight Hall Student Union Board of Directors (SUB)
```

Comment:
let's keep an eye out if a non hard coded source of clubs becomes readily available to us